contracts/src/v0.8/automation/dev/v2_3/AutomationRegistry2_3.sol

```diff
@@ -311,12 +311,16 @@ contract AutomationRegistry2_3 is AutomationRegistryBase2_3, OCR2Abstract, Chain
     // remove any old signer/transmitter addresses
     address signerAddress;
```

Comment:
nitpick, but I feel like this is unnecessary. I think we could just do...

```sol
delete s_signers[s_signersList[i]];
```

Reply:
done